### ISSUE TITLE
Training not making progress

### ISSUE THREAD

**Issue by geoffreywwang** [dariofuoli/RLSP#7]
I'm trying to train the pytorch model from scratch on the REDs dataset. I've modified the dataset/dataloader to support loading and cropping the videos, but the model doesn't seem to make any progress when training. Training loss hovers around 0.003 indefinitely and the validation PSNR hovers around 10.3. Parameters are unchanged from what's in the repository. Do you know what could be going wrong?

Thanks in advance!

**Comment by dariofuoli** [comment]
Hi,
I just ran the pytorch implementation with some dummy data, it learns as expected. My guess is, there might be some issue in your dataloader implementation?

Best,
Dario

**Comment by geoffreywwang** [comment]
Hi Dario,

Thanks for spending the time to help me look into this. It turned out to be exactly as you explained. I think the dataloader you use takes in floats ranging from 0-255 and not floats ranging from 0-1.

For anyone looking to create their own dataloader, I ended up loading in data with something like this:

```python3
import torch
from torchvision.io import read_image

# Reads in images as integer arrays with a range of 0-255
temp_lr = read_image('LOW_RES_FRAME_PATH')
temp_hr = read_image('HIGH_RES_FRAME_PATH')

# Converts the integer arrays to float arrays
lr_seq.append(temp_lr.to(dtype=torch.float32))
hr_seq.append(temp_hr.to(dtype=torch.float32))
```

-Geoffrey
